lib/Frontend/Frontend.cpp

```diff
@@ -497,58 +565,131 @@ void CompilerInstance::performSema() {
   if (options.DebugTimeExpressionTypeChecking) {
     TypeCheckOptions |= TypeCheckingFlags::DebugTimeExpressions;
   }
+  return TypeCheckOptions;
+}
+
+// Return true if had load error
```

Comment:
This doesn't belong here; it should be a proper doc comment at the declaration site.

Reply:
Done! Should all function doc comments appear at declaration site rather than definition site? What's the general rule?

Reply:
Yep, that's the rule. It's coming from "there should be one canonical place" + "if a function is used from multiple files, the definition site isn't a good choice".